public/js/controllers/index.js

```diff
@@ -164,6 +171,19 @@ angular.module('mean.system').controller('IndexController', [
       messageArray.push(message);
       $scope.notifications = messageArray;
       $scope.messageLength = messageArray.length;
+      console.log(messageArray)
     });
+
+    $scope.two = false;
+    $scope.one = true;
```

Comment:
Please can you provide insight as to what $scope.one and $scope.two are? The variable names are not descriptive enough so I can't tell what they are doing